releng/devkit.py

```diff
@@ -12,6 +12,7 @@
 import subprocess
 import sys
 import tempfile
+import argparse
```

Comment:
Would be good to move this right before `import codecs`, so the imports are kept in alphabetical order.

Reply:
Actually this import is already below so we should drop it here.